static/js/stream_edit_subscribers.js

```diff
@@ -185,9 +185,19 @@ function subscribe_new_users({pill_user_ids}) {
     }
 
     function invite_failure(xhr) {
-        const error = JSON.parse(xhr.responseText);
+        let error;
+        if (xhr.status === 0) {
```

Comment:
I don't think it's just network errors; the server throwing a 500 should also leave the input enabled.

I feel like the input should be disabled only if there's nothing there, and we would be better off having this logic do that simpler thing.

Reply:
> I feel like the input should be disabled only if there's nothing there, and we would be better off having this logic do that simpler thing.

Sounds good. Updated the PR for this.